test/cli/install/migration/complex-workspace.test.ts

```diff
@@ -44,27 +45,36 @@ test("the install succeeds", async () => {
   var subprocess = Bun.spawn([bunExe(), "reset.ts"], {
     env: bunEnv,
     cwd,
-    stdio: ["ignore", "ignore", "ignore"],
+    stdio: ["ignore", "pipe", "pipe"],
```

Comment:
this should just be `inherit` no?

Reply:
that works too but given they were ignore before I assumed it'd be noise on success